addons/keda/outputs.tf

```diff
@@ -0,0 +1,15 @@
+output "service_account" {
```

Comment:
This can be removed, as we are not using `irsa_config` which creates serviceAccount with mentiond value name.